src/components/powerInfoPage/chart.tsx

```diff
@@ -0,0 +1,1784 @@
+import React, { useRef, useEffect, useState } from 'react';
+import * as d3 from 'd3';
+import dayjs from 'dayjs';
+import classNames from 'classnames';
+
+interface IApiData {
+  Consume: number;
+  Date: string;
+  Demand: number;
+  ESS: number;
+  EV: number;
+  Generate: number;
+  PV: number;
+  WT: number;
+}
+
+interface IProps {
+  mode: string;
+  date: Date;
+}
+
+interface IPadding {
```

Comment:
It seems like this `interface` didn't been used. Please delete it if it's useless.

Reply:
Done. Thanks!